CMakeLists.txt

```diff
@@ -10,8 +10,8 @@ set(OPEN62541_CPP open62541cpp)
 
 ## build settings - try and match the c library version
 set(VERSION_MAJOR 1)
-set(VERSION_MINOR 2)
-set(VERSION_PATCH 777)
+set(VERSION_MINOR 3)
+set(VERSION_PATCH 14)
```

Comment:
This is the newest version of version 1.3.*, there also exists version v1.4.6, but I thought maybe this one would be more stable. 🤷